pkg/kv/kvpb/BUILD.bazel

```diff
@@ -8,21 +8,17 @@ load(":gen.bzl", "batch_gen")
 go_library(
     name = "kvpb",
     srcs = [
-        ":gen-batch-generated",  # keep
-        ":gen-errordetailtype-stringer",  # keep
-        ":gen-method-stringer",  # keep
         "ambiguous_result_error.go",
         "api.go",
-        # DRPC protobuf file (api_drpc.pb.go) is currently generated manually.
-        # TODO (chandrat): Remove this once DRPC protobuf generation is
-        # integrated into the build process.
-        "api_drpc_hacky.go",
```

Comment:
Hmm, I'm not understanding where exactly we added the DRPC stuff to replace this.

Reply:
Right, it's auto-generated and it appears in the tree like this:
```
kvpb
├── ambiguous_result_error.go
├── api_drpc.pb.go
├── api_test.go
├── api.go
├── api.pb.go
```

The PR that auto-generates that is already merged so it must be appearing in everyone's repo.